src/config/external-applications.md

```diff
@@ -82,3 +82,26 @@ example, to build the control package, you must install the `openblas-devel`,
 
 To use MATLAB's help browser, live scripts, add-on installer, and simulink,
 install the `libselinux` package.
+
+## Steam
+
+Steam can be installed either via a native package, which requires [enabling the
+"nonfree" repository](../xbps/repositories/index.md###nonfree), or via
+[Flatpak](##flatpak). Dependency listings for different platforms and
+troubleshooting information for the native package can be found in its
+[Void-specific documentation file](./package-documentation/index.html), so the
+remainder of this section will deal with potential issues faced by Flatpak
+users.
+
+If you are using a different drive to store you game library run the following
+commmand, where `<storage>` is the mount point of your drive:
+
+`$ flatpak override com.valvesoftware.Steam --filesystem=<storage>`
```

Comment:
```suggestion
If you are using a different drive to store you game library, the `--filesystem` option from [flatpak-override(1)](https://man.voidlinux.org/flatpak-override.1) can prove useful.
```

If you think an example command is really important, it should be a full code block, using three \`\`\`.

Reply:
No, I think your suggestion with a link to the man page is better. I'll include it in my next push.